src/quoi-feur.ts

```diff
@@ -0,0 +1,38 @@
+import type { Message } from 'discord.js';
+import { config } from './config';
+
+const quoiDetector = new RegExp(/\b\s*[qQ][uU][oO][iI]\s*[.,!?]*\s*$/i);
+const ONE_MINUTE = 60000;
+
+const reactWithFeur = async (message: Message) => {
+  await message.react('🇫');
+  await message.react('🇪');
+  await message.react('🇺');
+  await message.react('🇷');
+};
+
+const reactWithCoubeh = async (message: Message) => {
+  await message.react('🇨');
+  await message.react('🇴');
+  await message.react('🇺');
+  await message.react('🇧');
+  await message.react('🇪');
+  await message.react('🇭');
+  await message.react('🔇');
+
+  message.member?.roles.add(config.discord.mutedRoleId);
+  setTimeout(() => {
```

Comment:
What if the bot gets shutdown while someone has this role, he won't be able to recieve the reaction anymore.

I think you need to clear this on startup, (or change the way it works with the key-value store).